website/source/docs/providers/aws/d/caller_identity.html.markdown

```diff
@@ -32,4 +40,6 @@ There are no arguments available for this data source.
 
 ## Attributes Reference
 
-`account_id` is set to the ID of the AWS account.
+`account_id` - The AWS Account ID number of the account that owns or contains the calling entity.
```

Comment:
I guess this wasn't a list before because there was only one item, but our usual style is for the exported attributes to be a bullet list using `*`.